client/src/components/lower-info-area/name-build-sub-components/example-name.vue

```diff
@@ -67,30 +53,26 @@ import { Component, Vue } from 'vue-property-decorator'
 export default class ExampleName extends Vue {}
 </script>
 
-<style scoped lang="sass">
-#example-name
-  p
-    font-size: 14px
-    margin-bottom: 0
-
-.name-chip
-  border: 1px dashed $p-blue-text
-  background-color: $grey-1
-  font-size: 36px
-  font-weight: bold
-  padding: 1.5rem
-
-.text-content
-  text-align: center
+<style scoped lang="scss">
+@import "@/assets/scss/base.scss";
 
-.text-divider-1
-  margin-left: 6.5rem
-  border-left: 1px solid rgba(0, 0, 0, 0.12)
-  height: 2rem
+#example-name {
+  p {
+    font-size: 14px;
```

Comment:
Same as above but `.875rem`. 
I seem really focused on font sizes here 😆

Reply:
You are correct. I missed these :)